MetaMask Plugin/How to create a MetaMask Plugin

```diff
@@ -0,0 +1,26 @@
+Introduction
+
+1. Metamask plugin what’s new? 
+2. How this will improve or advantages over the extension.
+
+
+3.1 BreakDown of new Plugin APIs of Metamask. (Short Video/ Writeup)
```

Comment:
```suggestion
3.1 Break down of new the Plugin APIs of Metamask. (Short Video/ Writeup)
```

Any more detail here?

Reply:
https://workflowy.com/s/plugin-concepts-coll/BmH5IUfRIHa6EmHJ

I will refer to this. For different plugin related concepts, and will give a breakdown of the plugin, and its use case and how a developer can hack on it. Just the high-level approach.